<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtWDWRegistroDeArticulos</class>
 <widget class="QDialog" name="qtWDWRegistroDeArticulos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDP || Registro De Articulos</string>
  </property>
  <widget class="QPushButton" name="qtBTNAgregarArticulo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Articulo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qtBTNLimpiar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qtTXTNombre">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qtTXTPrecio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="qtSPNSpinCantidad">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
